torch_xla/csrc/runtime/computation_client.h

```diff
@@ -350,6 +353,17 @@ class ComputationClient {
   // the local devices will be waited for.
   virtual void WaitDeviceOps(const std::vector<std::string>& devices) = 0;
 
+  // Check whether the XlaCoordinator has been initialized.
+  virtual bool CoordinatorInitialized() const = 0;
```

Comment:
Do these need to be virtual? It looks like the implementations below don't depend on the underlying runtime client.

Reply:
The XlaCoordinator depends on PJRT, so I kept it separate.